test/LibraryManager.IntegrationTest/TestSolution/WebApplication1/WebApplication1.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Please give this project (and it's folder) a meaningful name.  Something like TestProjectCore20 or something to indicate what it is.

Reply:
Will do